auth_brute_force/README.rst

```diff
@@ -0,0 +1,114 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+===============================================================
+Tracks Authentication Attempts and Prevents Brute-force Attacks
+===============================================================
+
+This module registers each request done by users trying to authenticate into
+Odoo. If the authentication fails, a counter is increased for the given remote
+IP. After after a defined number of attempts, Odoo will ban the remote IP and
+ignore new requests.
+This module applies security through obscurity
+(https://en.wikipedia.org/wiki/Security_through_obscurity),
+When a user is banned, the request is now considered as an attack. So, the UI
+will **not** indicate to the user that his IP is banned and the regular message
+'Wrong login/password' is displayed.
+
+This module realizes a call to a web API (http://ip-api.com) to try to have
+extra informations about remote IP.
+
+Known issue / Roadmap
+---------------------
+The ID used to identify a remote request is the IP provided in the request
+(key 'REMOTE_ADDR').
+Depending of server and / or user network configuration, the idenfication
+of the user can be wrong, and mainly in the following cases:
+
+* if the Odoo server is behind an Apache / NGinx proxy without redirection,
+  all the request will be have the value '127.0.0.1' for the REMOTE_ADDR key;
+* If some users are behind the same Internet Service Provider, if a user is
+  banned, all the other users will be banned too;
```

Comment:
But this is expected to be the usual case, isn't it?
I'm a little out of step here, be shouldn't we also look into `X-Forwarded-For`?